pyvista/core/dataset.py

```diff
@@ -586,6 +586,29 @@ def translate(self, xyz: Union[list, tuple, np.ndarray]):
         """
         self.points += np.asarray(xyz)
 
+    def scale(self, xyz: Union[list, tuple, np.ndarray]):
+        """Scale the mesh.
+
+        Parameters
+        ----------
+        xyz : scale factor list or tuple or np.ndarray
+            Length 3 list, tuple or array.
+
+        """
+        self.points *= np.asarray(xyz)
+
+    def flip_x(self):
+        """Flip mesh about the x-axis."""
+        self.points[:, 0] *= -1.0
+
+    def flip_y(self):
+        """Flip mesh about the y-axis."""
+        self.points[:, 1] *= -1.0
+
+    def flip_z(self):
+        """Flip mesh about the z-axis."""
```

Comment:
Let's also add docstring examples here with a nice plot.  They'll show up in the new docbuild.

Reply:
@akaszynski Thank you for your comment. I added examples to docstring.
```python
import pyvista
from pyvista import examples
pl = pyvista.Plotter(shape=(1, 2))
pl.subplot(0, 0)
pl.show_axes()
pl.show_grid()
mesh1 = examples.download_teapot()
_ = pl.add_mesh(mesh1)
pl.subplot(0, 1)
pl.show_axes()
pl.show_grid()
mesh2 = mesh1.copy()
mesh2.scale([10.0, 10.0, 10.0])
_ = pl.add_mesh(mesh2)
pl.show(cpos="xy")
```
![test1](https://user-images.githubusercontent.com/7513610/128609932-c0e0ad0c-1398-489d-af26-69bb44669c38.png)
```python
import pyvista
from pyvista import examples
pl = pyvista.Plotter(shape=(1, 2))
pl.subplot(0, 0)
pl.show_axes()
mesh1 = examples.download_teapot()
_ = pl.add_mesh(mesh1)
pl.subplot(0, 1)
pl.show_axes()
mesh2 = mesh1.copy()
mesh2.flip_x()
_ = pl.add_mesh(mesh2)
pl.show(cpos="xy")
```
![test2](https://user-images.githubusercontent.com/7513610/128609981-f69e08d7-f6e9-43f0-918d-38c2975a3a19.png)